_extras/learner-prerequisites.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Learner Prerequisites
+---
+
+This page collects questions that learners can answer in order to judge if they 
+bring all prerequisites to the course.
+
+# Required Pre-Knowledge
+
+- **Basic Shell** – Navigating directories, Copy/Moving, writing shell scripts, 
+using the environment, using wildcards
+
+# Pre-Workshop Survey
```

Comment:
A proper Markdown file contains only one top-level header (`#`). Consider using `{: .prereq}` blocks for these two?

Reply:
see 80ea80c